staging/src/k8s.io/client-go/util/keyutil/key.go

```diff
@@ -0,0 +1,348 @@
+/*
```

Comment:
I would have named the package as "key" instead of "keyutil", just like the "cert" package is not named as "certutil".

Reply:
My concern is that `key` has a much higher likelihood of name collision with types/vars.
You can see that https://github.com/kubernetes/kubernetes/search?q=k8s.io%2Fclient-go%2Futil%2Fcert+certutil&unscoped_q=k8s.io%2Fclient-go%2Futil%2Fcert+certutil `cert` package is frequently aliased to `certutil` for this reason.